test/JDBC/expected/BABEL-IDENTITY.out

```diff
@@ -1208,7 +1208,7 @@ Index Only Scan using babel_3384_test_pkey on babel_3384_test
 
 ~~START~~
 text
-Babelfish T-SQL Batch Parsing Time: 0.159 ms
+Babelfish T-SQL Batch Parsing Time: 0.154 ms
```

Comment:
Can't these values change?

Reply:
Yes they can but the test framework ignores the Batch Parsing time when taking the diff. See: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_5_X_DEV/test/JDBC/src/test/java/com/sqlsamples/TestQueryFile.java#L380